src/lib/style/components/_entity-metadata.scss

```diff
@@ -42,17 +43,6 @@
   }
 }
 
-.MetadataTable {
-  width: 100%;
-  margin: 0px;
-  &__Row {
-    &__Key {
-      font-weight: bold;
-      padding: 6px 0px;
-    }
-
-    &__Value {
-      padding: 6px 0px;
-    }
-  }
+.FluidModal.EntityMetadata:not(.isInEditMode) .modal-dialog {
+  max-width: 600px;
 }
```

Comment:
We don't want the modal to take up the entire screen when we're just showing the metadata or annotations. Should only use the full screen when in edit mode.